components/mediation-ui/org.wso2.carbon.mediation.security.ui/src/main/resources/web/mediation_secure_vault/js/properties.js

```diff
@@ -49,26 +49,21 @@ function setProperty() {
 				reason += validateEmptySV(
 						document.getElementById('propName'),
 						org_wso2_carbon_mediation_secure_vault_ui_jsi18n["property.name"]);
-				// if (reason == "") {
-				// reason
-				// +=validateEmptySV(document.getElementById('propValue'),
-				// org_wso2_carbon_mediation_secure_vault_ui_jsi18n["property.value"]);
-				// }
-              
-
-				if (reason == "") {
+				if (reason === "") {
 					reason += validateForInputSV(
 							document.getElementById('propName'),
 							org_wso2_carbon_mediation_secure_vault_ui_jsi18n["property.name"]);
 				}
-				if (reason == "") {
-					reason += validateForInputSV(
+				if (reason === "") {
+                	reason +=validateEmptySV(document.getElementById('propValue'),
```

Comment:
It seems we have used tabs instead of spaces which mess with the indentation. Shall we use space instead of tabs?